### PR TITLE
Script to sync workload identity bindings

### PR DESCRIPTION
**Pull request by fejta**

### CODE REVIEW

File: cluster/bind-service-accounts.sh [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+      echo "NOOP: $membermsg has $rolemsg access to $gsa"
+      continue
+    fi
+
+    case "$add" in
+      y*|Y*)
+      (
+        set -o xtrace
+        "$gcloud" iam service-accounts \
+          --project "$gsa_project" \
```
Comment (chases2) [inline cluster/bind-service-accounts.sh:57]: I thought `${gsa_project}` was preferred in bash?

**Review by chases2 — APPROVED** [review]
/hold

File: cluster/bind-service-accounts.sh (hunk 1 above)
Comment (fejta) [reply cluster/bind-service-accounts.sh:57]: Technically correct: https://google.github.io/styleguide/shellguide.html#quoting but I think shellcheck has more sane guidelines: https://github.com/koalaman/shellcheck I will happily follow it here if someone sets it up

Comment (fejta) [reply cluster/bind-service-accounts.sh:57]: https://github.com/kubernetes/kubernetes/blob/master/hack/verify-shellcheck.sh

**Pull request merged**